### PR TITLE
Fix #17328 - Move NFT settings to Experimental

### LINKED ISSUE #17328: NFTs: Settings required to import aren't logically communicated
### Describe the bug

Nidhi and I couldn't figure out why she wasn't having a problem importing an NFT bought from OpenSea.  It turned out the problem was that she didn't have "Autodetect tokens" on.

We need to better figure out the settings situation so the user knows exactly what needs to be turned on to get NFTs working properly.  

Additionally, instead of just showing a "cannot import NFT" message, we should give the user a link to settings if the required settings aren't enabled.

NFTSettings

### Steps to reproduce

1.  Import a wallet that owns an NFT on MainNet
2.  Enable "Settings/Privacy -> Enable NFT Detection" and "Experimental -> Enable OpenSea API"
3. Try to import the NFT -- you'll get a message saying that ownership can't be verified
4. Enable "Settings/Privacy -> Enable Token Detection"
5. Try to import the NFT -- the import works

This is a really bad experience.

### Error messages or log output

_No response_

### Version

10.23

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

* Fixes #17328

This moves the "Autodetect NFTs" setting block to the Experimental tab.

## Screenshots/Screencaps

ExpSettings

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by SaraCheikh** [comment]
@darkwing UI can def look less chaotic 😂 I [redesign this screen in both pop up and full screen. ](https://www.figma.com/file/cAHj7i7USWsMkzDsePN7zy/Settings-%3E-Security-%26-Privacy?node-id=4426%3A5044&t=SUejH7Ks8wa65jkm-1) some thoughts:

- We don't need the level "on/off" in the toggle, the icon is already self explanatory
-Enable Opensea API and Autodetect NFTs are both associated, so if Autodetect NFTs is off by default, Opensea API would also be off. If they both have the same behaviour, why not unify them in the same setting? We don't have a separate setting for "Enable Coingecko API" we just mention in "Show balance and token price checker" that we use their API.

**Review by brad-decker — DISMISSED** [review]

**Review by vthomas13 — DISMISSED** [review]

File: ui/pages/settings/experimental-tab/experimental-tab.component.js [hunk 1]
```diff
@@ -62,41 +59,77 @@ export default class ExperimentalTab extends PureComponent {
...
-          <span>{t('enableOpenSeaAPI')}</span>
-          <div className="settings-page__content-description">
-            {t('enableOpenSeaAPIDescription')}
+      <>
+        <div ref={this.settingsRefs[0]} className="settings-page__content-row">
+          <div className="settings-page__content-item">
+            <span>{t('enableOpenSeaAPI')}</span>
+            <div className="settings-page__content-description">
+              {t('enableOpenSeaAPIDescription')}
+            </div>
```
Comment (NidhiKJha) [inline ui/pages/settings/experimental-tab/experimental-tab.component.js:68]: Nit: Can we use more semantic tag than div here?

**Review by NidhiKJha — COMMENTED** [review]
LGTM! Just a couple of nits.

File: ui/pages/settings/experimental-tab/experimental-tab.component.js [hunk 2]
```diff
@@ -62,41 +59,77 @@ export default class ExperimentalTab extends PureComponent {
...
-                setOpenSeaEnabled(!value);
-              }}
-              offLabel={t('off')}
-              onLabel={t('on')}
-            />
+        <div ref={this.settingsRefs[1]} className="settings-page__content-row">
+          <div className="settings-page__content-item">
+            <span>{t('useCollectibleDetection')}</span>
+            <div className="settings-page__content-description">
+              {t('useCollectibleDetectionDescription')}
```
Comment (NidhiKJha) [inline ui/pages/settings/experimental-tab/experimental-tab.component.js:99]: Nit: Not sure if we should use div for description here wdyt

File: ui/pages/settings/experimental-tab/experimental-tab.component.js (hunk 1 above)
Comment (darkwing) [reply ui/pages/settings/experimental-tab/experimental-tab.component.js:68]: Let's address this in a separate issue; unfortunately it's consistent with the rest of the `Settings` pages.

**Review by pedronfigueiredo — APPROVED** [review]

**Review by vthomas13 — APPROVED** [review]

**Pull request merged**